src/AccessibilityInsights.Win32/Win32Enums.cs

```diff
@@ -188,7 +185,7 @@ public enum FontPitchAndFamily : UInt32
         DEFAULT_PITCH = 0,
         FIXED_PITCH = 1,
         VARIABLE_PITCH = 2,
-        FF_DONTCARE = (0 << 4),
+        // FF_DONTCARE = (0 << 4),
```

Comment:
Any particular reason to comment this out but delete the others?

Reply:
Yes. FF_DONTCARE actually has a distinct meaning, while the others are merely duplicates. I kept it commented just in case we ever need it (unlikely, for sure...)